src/commands/HBonds.cpp

```diff
@@ -0,0 +1,206 @@
+                                selection of size 1.
+                                [default: atoms: not type H]
+  -p <par>, --parameters=<par>  parameters to use for the hydrogen bond. <par>
+                                format is <d:α> where 'd' is the donor-acceptor 
+                                maximum distance in angstroms and 'α' is the
+                                donor-acceptor-hydrogen maximum angle in degrees.
+                                [default: 3.0:30.0]
+)";
+
+static HBonds::Options parse_options(int argc, const char* argv[]) {
+    auto options_str = command_header("hbonds", HBonds().description()) + "\n";
+    options_str += OPTIONS;
+    auto args = docopt::docopt(options_str, {argv, argv + argc}, true, "");
+
+    HBonds::Options options;
+    options.trajectory = args.at("<trajectory>").asString();
+    options.guess_bonds = args.at("--guess-bonds").asBool();
+
+    options.selectionAcceptor = args.at("--selection_acceptor").asString();
+    options.selectionDonor = args.at("--selection_donor").asString();
+
+    if (args.at("--output")){
+        options.outfile = args.at("--output").asString();
+    } else {
+        options.outfile = options.trajectory + "_hb.dat";
+    }
+
+    if (args.at("--steps")) {
+        options.steps = steps_range::parse(args.at("--steps").asString());
+    }
+
+    if (args.at("--format")){
+        options.format = args.at("--format").asString();
+    }
+
+    if (args.at("--topology")){
+        if (options.guess_bonds) {
+            throw CFilesError("Can not use both '--topology' and '--guess-bonds'");
+        }
+        options.topology = args.at("--topology").asString();
+    }
+
+    if (args.at("--topology-format")){
+        if (options.topology == "") {
+            throw CFilesError("Useless '--topology-format' without '--topology'");
+        }
+        options.topology_format = args.at("--topology-format").asString();
+    }
+
+    if (args.at("--cell")) {
+        options.custom_cell = true;
+	options.cell = parse_cell(args.at("--cell").asString());
+	}
+
+    options.distance_parameter = 3.0;
+    options.angle_parameter = 30.0;
+    if (args.at("--parameters")) {
+	auto splitted = split(args.at("--parameters").asString(), ':');
+	if (splitted.size() == 2) {
+	    options.distance_parameter = stod(splitted[0]);
+	    options.angle_parameter = stod(splitted[1])*pi/180;
+	} else {
+            throw CFilesError(
+                "custom parameters should be specified as 'd:α'"
+            );
+    	}
+    }	
+
+    return options;
+}
+
+
+std::string HBonds::description() const {
+    return "compute hydrogen bonds network";
+}
+
+int HBonds::run(int argc, const char* argv[]) {
+    auto options = parse_options(argc, argv);
+
+    selectionAcceptor_ = Selection(options.selectionAcceptor);
+    if (selectionAcceptor_.size() != 2) {
+        throw CFilesError("Can not use a selection for acceptors with size different than 2.");
```

Comment:
You require `bonds:` selection in the documentation, but the selection could be of kind `pairs:` too here.